plutus-playground-client/test/BridgeTests.purs

```diff
@@ -1,24 +1,45 @@
 module BridgeTests
-       ( all
+       ( all, k
        ) where
 
 import Prelude
 
+import AjaxUtils (ajaxSettings)
+import Control.Monad.Aff.AVar (AVAR)
+import Control.Monad.Eff (Eff)
 import Control.Monad.Eff.Class (class MonadEff, liftEff)
+import Control.Monad.Eff.Console (CONSOLE)
 import Control.Monad.Eff.Exception (EXCEPTION)
 import Control.Monad.Eff.Random (RANDOM)
-import Data.Argonaut.Generic.Aeson (decodeJson)
 import Data.Argonaut.Parser (jsonParser)
 import Data.Either (Either(..))
 import Data.Generic (class Generic)
+import Data.List (List)
+import Data.List.Types (NonEmptyList)
 import Language.Haskell.Interpreter (CompilationError)
 import Node.Encoding (Encoding(UTF8))
 import Node.FS (FS)
 import Node.FS.Sync as FS
-import Playground.API (CompilationResult, EvaluationResult)
+import Playground.API (CompilationResult, EvaluationResult, KnownCurrency, TokenId)
+import Servant.PureScript.Settings (SPSettingsDecodeJson_(..), SPSettings_(..))
 import Test.Unit (TestSuite, Test, failure, success, suite, test)
+import Test.Unit.Console (TESTOUTPUT)
+import Test.Unit.Main (runTest)
 import Type.Proxy (Proxy(..))
 
+k :: forall t57.
```

Comment:
what's this? `k` and `t57` ?

Reply:
Oops, shouldn't have been committed. Thanks. :-D